learn/tanssi/services-payment.md

```diff
@@ -0,0 +1,45 @@
+- **Node operators** - bearing with the responsibility of keeping their hardware and software configuration in optimal conditions, they are rewarded with Tanssi tokens for executing transactions and producing blocks on behalf of Tanssi and Tanssi appchains
+
+The [block production as a service](/learn/tanssi/technical-features/#block-production-as-a-service){target=\_blank} mechanism randomly selects a subset of a decentralized set of nodes and assigns them to provide services to Tanssi and the Tanssi-powered appchains for a limited period of time (session). Once the session ends, the Tanssi protocol will again select the set of block producers needed to serve Tanssi and all the active Tanssi appchains and re-assign them.
+
+All this operation carries costs that must be covered by the appchains that want to obtain the services. This article covers the general aspects of those costs and associated service payments.
+
+## Costs to Cover  {: #costs-to-cover }
+
+There are three main costs associated with block production as a service that any appchain must cover using Tanssi tokens to deploy successfully and get the block production services:
+
+- **Registration deposit** - is the initial deposit that is locked from the account that signs the appchain registration transaction
+- **Block producers assignment** - every time the Tanssi protocol assigns block producers, which happens once per session, a fixed fee is charged. This discourages a faulty appchain (whose runtime logic fails to produce valid transitions and, therefore, a valid block) from requesting being assigned nodes that will not be able to produce valid blocks
+- **Block production** - appchains need to pay for each block that is produced on their behalf. Since the protocol selects and assigns the block producers on a per-session basis, appchains must have enough funds to cover all the blocks to be produced in an entire session to be served
+
+On Dancebox, the Tanssi testnet, the current configuration is the following:
+
+=== "Dancebox"
+    |         Variable          |                                   Value                                   |
+    |:-------------------------:|:-------------------------------------------------------------------------:|
+    |   Registration deposit    |         {{ networks.dancebox.costs.registration_deposit }} DANCE          |
+    | Block producer assignment | {{ networks.dancebox.costs.cost_per_assignment }} microDANCE per session |
+    |     Block production      |    {{ networks.dancebox.costs.cost_per_block }} microDANCE per block     |
+
+To ensure block production in the next session, the total balance must be at least enough to cover the block producer assignment cost plus the cost to produce the {{ networks.dancebox.session.blocks }} blocks that comprise an entire session.
```

Comment:
how is this paid for? i'm assuming in the dapp?

Reply:
Yup, in the dapp there's a "top-up" section (the tipping mechanism is not yet there though)